docs/navigation.html

```diff
@@ -76,6 +76,7 @@
   <li {% if page.path == 'Pagination.md' %} class="active" {% endif %}><a class="nav-link" href="./Pagination.html"><code>&lt;Pagination&gt;</code></a></li>
   <li {% if page.path == 'SortButton.md' %} class="active" {% endif %}><a class="nav-link" href="./SortButton.html"><code>&lt;SortButton&gt;</code></a></li>
   <li {% if page.path == 'SelectColumnsButton.md' %} class="active" {% endif %}><a class="nav-link" href="./SelectColumnsButton.html"><code>&lt;SelectColumnsButton&gt;</code></a></li>
+  <li {% if page.path == 'Count.md' %} class="active" {% endif %}><a class="nav-link" href="./Count.html"><code>&lt;Count&gt;</code></a></li>
```

Comment:
Don't you need to add them to the Reference page too?

Reply:
You're right, I forgot! Fixed in the last commit.